### ISSUE TITLE
Move to Zstd 1.5.5 to avoid corruption in high compression scenarios

### ISSUE THREAD

**Issue by z3cko** [rustic-rs/rustic#753]
Zstd 1.5.5 has been [released already on 2023-04-05](https://www.phoronix.com/news/Zstd-1.5.5), but when installing rustic-rs via cargo, the installed Zstd version seems to be much older ("Compiling zstd v0.12.3+zstd.1.5.2"). Is there something we can do about this? This is very concening, specifically because there was a file corruption bug identified in Zstd 1.5.4, hence we want to make sure to be on the latest version of Zstd.

**Comment by aawsome** [comment]
@simonsan I agree that we should make a new patch version. Unfortunately we are currently not in a state where we can release - as the rustic_core redesign has not yet finished. So I propose to finish that soon and then release a new version.
